__main__.py

```diff
@@ -4,43 +4,99 @@
 Usage: run the exerciser with a variety of options
 """
 
-from pathlib import Path
 import argparse
-from datetime import datetime
 import sys
 import os
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
 from src import general
 
+
 def parse_cla() -> argparse.Namespace:
     """
     Usage: command line argument parser
     @return: parsed arguments in argparse.Namespace object
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("test_name",
-                        type=str,
-                        metavar="test_name",
-                        choices=["sleep_test", "checksum_test"],
-                        help="name of test to be run by exerciser")
+    parser.add_argument(
+        "tests",
+        nargs="*",
+        default=[],
+        help="specify the test/s to run in a space seperated list. if no tests "
+        + "are listed, the exerciser will run all available tests.",
+    )
+
+    parser.add_argument(
+        "-f",
+        "--flush-all",
+        action="store_true",
+        dest="flush_all",
+        help="deletes all subdirectories in the working dir. if this option is used with "
+        + "-w, it will flush the contents of that dir rather than the default working dir.",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--flush-by-date",
+        metavar="YYYY-MM-DD_hh-mm",
+        dest="flush_by_date",
+        help=f"deletes all subdirectories in the working dir older than the arg value. "
+        + "if this option is used with "
+        + "-w, it will flush the contents of that dir rather than the default working dir. "
+        + "argument value must include a year, but all other date specifications are optional. "
+        + "if included, they must follow the YYYY-MM-DD_hh-mm format. for example, the following "
+        + "are valid arg values (seperated by commas): 2024, 2024-07, 2024-07-30_02, "
+        + "2024-07-31_14-53. but the following are invalid args: 202, 2024_14, 07-31, 2024-",
```

Comment:
Consider the following description:
`Removes all exerciser execution subdirectories in the specified working directory older than the specified date/time. Note: date/time can be any level of specificity from just YYYY to full YYYY-MM-DD_hh-mm format`